elasticdl/proto/elasticdl.proto

```diff
@@ -94,13 +94,27 @@ message ReportVersionRequest {
   int32 model_version = 1;
 }
 
+
+message GetRankRequest {
+  int32 worker_id = 1;
+}
+
+
+message GetRankResponse {
+  int32 rank_id = 1;
+  int32 world_size = 2;
+  int32 rendezvous_id = 3;
+  int32 rendezvous_port = 4;
+}
+
 service Master {
   rpc get_task(GetTaskRequest) returns (Task);
   rpc report_evaluation_metrics(ReportEvaluationMetricsRequest)
       returns (google.protobuf.Empty);
   rpc report_task_result(ReportTaskResultRequest)
       returns (google.protobuf.Empty);
   rpc report_version(ReportVersionRequest) returns (google.protobuf.Empty);
+  rpc get_comm_rank(GetRankRequest) returns (GetRankResponse);
```

Comment:
get_communication_rank?

Reply:
We can use the abbreviation "comm" to stand for "communicator" like the method [MPI_Comm_rank](https://www.mpich.org/static/docs/v3.2/www3/MPI_Comm_rank.html) in MPI.